docs/dev/cas.md

```diff
@@ -0,0 +1,64 @@
+# How to test CAS as a developer without a server
+
+The [django-mama-cas](https://github.com/jbittel/django-mama-cas) project is an
+easy to run CAS implementation built on top of Django.
+
+## Prerequisites
+
+1. Create a new virtualenv: `python3 -m venv <your virtualenv>`
+2. Activate your virtualenv: `source /path/to/your/virtualenv/bin/activate`
+3. Install Django and django-mama-cas:
+   ```
+   python -m pip install "django<3" "django-mama-cas==2.4.0"
+   ```
+4. Create a Django project in the current directory:
+   ```
+   django-admin startproject cas_test .
+   ```
+5. Follow the [install directions](https://django-mama-cas.readthedocs.io/en/latest/installation.html#configuring) for django-mama-cas
```

Comment:
I tried following these instructions and got stuck here. The linked instructions ask me to run `migrate`, but I'm not sure what this means. There certainly isn't a binary named `migrate` in my path.

Then I tried `python manage.py migrate`, thinking that might be it, but there's no file called `manage.py`.

What am I missing? :)

Reply:
There should be a file called `manage.py` after running the step above this (`django-admin startproject ...`)

Reply:
Ah, that works! Would probably help if I didn't test this in such a cluttered folder.

Reply:
I would have liked the directions to be simpler, but...this seemed easier than a "full" CAS setup.